tests/test_hf_upload.py

```diff
@@ -0,0 +1,12 @@
+import hf_upload
```

Comment:
currently in the CI, this causes an import error, is there a way perhaps to make this import work? One option could be to move this inside the library?